extensions/amp-user-notification/amp-user-notification.md

```diff
@@ -121,11 +121,14 @@ will be passed in future requests to data-show-if-href)
 #### JSON Fields
 
 - `elementId` (string) - The HTML id used on `amp-user-notification` element.
```

Comment:
Google style and general style in JS is to do strict camelCase as in theHtmlDom.